### DISCUSSION TITLE
4/16 - Discussion with Prof

### DISCUSSION THREAD

**Discussion by willschenk** [ellenguyen/CIS4496_EY#15] [General]
DEMO FEEDBACK 04/16: 

1. Definition correction: Hierarchical model (like a binary tree) -> only predict 1 sub-branch vs Sequential model (consider all aspects) -> type then damage

2. Class imbalance: create more synthetic images (slight rotation, flipping, etc.) for minority classes

3. Model development:
- other than using pre-trained model, try adjusting the params (drop-out)
- model look at the weights to drop out some irrelevant layers that are contributing noise

4. if Sequential approach: vectorize the bounding boxes -> calculate euclidean distance -> overlapping bounding boxes AND vectorize labels using 0s 1s to detect all 4 classes at once

5. For fine-tuning with cars detection: if just based on distribution of cars, hard to tell if it's more likely to be residential or commercial. even human eyes cant tell lmao
